action/cloud_token: added cloud-token to initiate atlas-login (#4)
ariga/setup-atlas: README.md

        with:
            version: "vX.Y.Z"
```
### cloud-token

The Atlas Cloud token to use for authentication. Must be passed as a secret.
```yaml
      - uses: ariga/setup-atlas@master
        with:

>             cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}

Author: missing ```

        with:
            version: "vX.Y.Z"
```

> ### Cloud Token
> 
> Some Atlas features require cloud integration, the `cloud-token` is used to authenticate with Atlas Cloud. More information about atlas cloud can be found [here](https://atlasgo.io/cloud/getting-started)

Reviewer: Use the actual param as used [here](https://github.com/ariga/atlas-sync-action#configuration)
```suggestion
### cloud-token

API token to authenticate to an Atlas Cloud organization. [Read more](https://atlasgo.io/cloud/getting-started)
```

> ### Cloud Token
> 
> Some Atlas features require cloud integration, the `cloud-token` is used to authenticate with Atlas Cloud. More information about atlas cloud can be found [here](https://atlasgo.io/cloud/getting-started)
> 
> - Note: You should be treating the cloud token as a secret. The below example uses github secrets for that
> ```yaml
>       - uses: ariga/setup-atlas@master
>         with:
>             cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}
> ```

Reviewer: See this as a reference: https://github.com/ariga/atlas-sync-action#cloud-token

```suggestion
### cloud-token

The Atlas Cloud token to use for authentication. Must be passed as a secret.
```yaml
      - uses: ariga/setup-atlas@master
        with:
            cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}
```
```
